docs/source/changelog.rst

```diff
@@ -60,7 +60,7 @@ Mobjects, Scenes, and Animations
 #. The :code:`Container` class has been made into an AbstractBaseClass, i.e. in cannot be instantiated.  Instead, use one of its children classes
 #. The ``TextMobject`` and ``TexMobject`` objects have been deprecated, due to their confusing names, in favour of ``Tex`` and ``MathTex``. You can still, however, continue to use ``TextMobject`` and ``TexMobject``, albeit with Deprecation Warnings constantly reminding you to switch.
 #. Add a :code:`Variable` class for displaying text that continuously updates to reflect the value of a python variable.
-
+#. :code:`VGroup` and :code:`VDict` now support printing the contained mobjects.
```

Comment:
**Suggestion** : Precise how it's printed (like, that the objects are printed in case of a VGroup.